crates/types/src/wots.rs

```diff
@@ -40,6 +66,14 @@ impl Wots256PublicKey {
     }
 }
 
+impl Deref for Wots160PublicKey {
+    type Target = [[u8; WOTS_SINGLE]; 160];
```

Comment:
this doesn't make sense to me. I thought that each WOTS key was

`M / D * W`
where
`M` = Message digest length
`D` = Digit width (also called window size)
`W` = per-digit pubkey width.

This seems to be an array of 160 20-byte values.

Reply:
fixed in e764b4da19b4921ca913e320edfd26d0bc6d8761
Thanks for pointing it out.